tests/js/fluid/html/index.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en">
+<head>
+    <title>Fluid invokeGlobalFunction test page...</title>
+    <script src="../../../../node_modules/infusion/src/lib/jquery/core/js/jquery.js"></script>
+    <script src="../../../../node_modules/infusion/src/framework/core/js/Fluid.js"></script>
+</head>
+<body>
+<p>This is content.</p>
+<script type="text/javascript">
+    var existingGlobalFunction = function () {
+        return true;
+    };
+
+    var existingGlobalFunctionWithArgs = function () {
+        return arguments.length;
+    };
+
+    var namespaced = {
+        fn: function(){ return true; }
```

Comment:
Done.  In moving this to another file, I ended up namespacing the previously global functions, which made the final check (which tested the use of deeper paths) redundant.